server/views/pages/manage-your-notifications.njk

```diff
@@ -12,7 +12,7 @@
 
   <main id="main-content" role="main">
     <h1 class="govuk-heading-l">Manage your notifications</h1>
-    <p class="govuk-body">You can be notified by email when this service has been updated with any change to your shift
+    <p class="govuk-body">You can be notified by email or text when this service has been updated with any change to your shift
```

Comment:
revert earlier change ( text notification feature was removed then added back in)